### ISSUE TITLE
Ambigous licensing

### ISSUE THREAD

**Issue by iimog** [PitchInteractiveInc/Phinch#56]
The README file states that the License is "BSD 2-Clause" which is consistent with the note on the phinch.org website, however there is also a LICENSE file which contains the GPL v3 License text.

Please replace the LICENSE file with a copy of the "BSD 2-Clause" license to remove ambiguity.

**Comment by hollybik** [comment]
Thanks for pointing this out - we'll get it fixed.

**Comment by iimog** [comment]
Any news on that? I'm planning to integrate Phinch into another MIT licensed project. This is no problem if Phinch is under the "BSD 2-Clause". The GPL v3 license on the other hand is not compatible with the MIT license.
It would help me a lot to know whether I can use Phinch to create awesome visualizations or if I have to look for an alternative.

**Comment by hollybik** [comment]
Yes, Phinch is under the BSD 2-Clause - the discrepancy is in the documentation of this LICENSE file. The correct license (BSD 2) is listed on our wiki - https://github.com/PitchInteractiveInc/Phinch

I'll note that we need to replace the GPL license text with the BSD 2 text...

**Comment by iimog** [comment]
Thank you @hollybik that is good news. So I will be able to use Phinch in my project. If I make improvements to Phinch I will definitely contribute them back to the main project.

Thanks a lot for this great project.
